dpgen/generator/run.py

```diff
@@ -2672,10 +2672,11 @@ def make_fp_abacus_scf(iter_index,
             if fp_params['basis_type'] == 'lcao':
                 assert('fp_orb_files' in jdata and type(jdata['fp_orb_files']) == list and len(jdata['fp_orb_files']) == len(fp_pp_files))
                 fp_orb_files = jdata['fp_orb_files']
-        if 'deepks_out_labels' in fp_params:
-            if fp_params['deepks_out_labels'] == 1:
-                assert('fp_dpks_descriptor' in jdata and type(jdata['fp_dpks_descriptor']) == str)
-                fp_dpks_descriptor = jdata['fp_dpks_descriptor']
+        dpks_out_labels = 0 if 'deepks_out_labels' not in fp_params else fp_params['deepks_out_labels']
```

Comment:
This line can be written as
```suggestion
        dpks_out_labels = fp_params.get('deepks_out_labels', 0)
```

Same for lines below.

Reply:
ok